File: 93/C.py
# encoding:utf-8

import copy
import numpy as np
import random

nums = [int(i) for i in input().split()]
ans = 0
even = 0

for i in range(3):
    if nums[i]%2 == 0:
        even+=1

if even==3:
    pass
elif even==2:
    for i in range(3):
        if nums[i]%2==0:
            nums[i]+=1
    ans+=1

elif even==1:
    for i in range(3):
        if nums[i]%2==1:
            nums[i]+=1
    ans+=1
else:
    pass

goal_num = max(nums)
ans += (3*goal_num-sum(nums))//2
print(ans)

File: 93/C.cpp
//encoding:utf-8
#include <iostream> //c++
#include<time.h> //時間にかんして
#include<stack> //スタックに関して　最新のデータから捨てる
#include<queue> //キューに関して　最古のデータから捨てる
#include<math.h>
#include<vector>
#include<algorithm> //sort とかにいる

#define ll long long
using namespace std;
int main(){
  vector<int>nums(3);
  int even=0;
  int ans=0;
  for(int i=0;i<3;i++){
    cin>>nums[i];
    if(nums[i]%2==0){
      even++;
    }
  }
  if(even==3){
  }
  else if(even==2){
    for(int i=0;i<3;i++){
      if(nums[i]%2==0){
        nums[i]+=1;
      }
    }
  ans++;
  }
  else if(even==1){
    for(int i=0;i<3;i++){
      if(nums[i]%2==1){
        nums[i]+=1;
      }
    }
  ans++;
  }
  else{}
  int max = 0;
  int sum = 0;
  for(int i=0;i<3;i++){
    sum += nums[i];
    if(max<nums[i]){
      max=nums[i];
    }

  }
  ans+=((3*max-sum)/2);

  cout<<ans<<endl;
  return 0;
}
